test/Infrastructure/LeanCode.Serialization.Tests/LeanCode.Serialization.Tests.csproj

```diff
@@ -1,5 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
+  <PropertyGroup>
+    <NoWarn>SYSLIB1040</NoWarn>
```

Comment:
Reason?

Reply:
It's because I'm using non-generic converters on some enums here, I'll be able to drop this once I drop those converters and related tests.

Reply:
One last question - why the non-generic variant generates this? [MSDN](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib1040-1049) has such a sparse description of the error that I can't really find out why it is raised in the first place. :)

Reply:
Docs seem to be outdated, the message I'm seeing is:

> CSC : error SYSLIB1040: The member 'LeanCode.Kratos.Model.IdentityState' has been annotated with 'JsonStringEnumConverter' which is not supported in native AOT. Consider using the generic 'JsonStringEnumConverter<TEnum>' instead.

Reply:
Thx!